examples/Demo/FluentUI.Demo.Client/Documentation/Components/Tooltip/FluentTooltip.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Tooltip
+route: /Tooltip
+---
+
+# Tooltip
+
+A `FluentTooltip` displays additional information about another component. The information is displayed above and near the target component.
+
+**Tooltip** is not expected to handle interactive content. If this is necessary behavior, an expand/collapse button + popover should be used instead.
+
+Hover or focus the buttons in the examples to see their tooltips.
+
+## Default
+
+{{ TooltipDefault }}
+
+## Customized
+The tooltip can be customized with a custom template,
+adapting the delay to `700 ms`, the position of the tooltip to `BelowStart` (including spacings to `20px` and `10px`)
+and the style "Inverted".
+
+Each time the tooltip is shown or hidden, a message is logged in the console. Click on the top/right button to open the **Console**.
+
+{{ TooltipCustomized }}
+
+## Using Tooltip Service Provider
+
+We advise you to always use this service when working with tooltips so all definitions will be generated with global options
+and will be written **at the end** of the HTML page to support different z-index levels.
+
+In the `Program.cs`, inject the Tooltip service with global options.
+You can also use the `Services.AddFluentUIComponents` method to register **all** required FluentUI services, including this one.
```

Comment:
For more convenience, you should use the `Services.AddFluentUIComponents()` method to register **all** required Fluent UI services (including this one).